Framework/PythonInterface/plugins/algorithms/AlignComponents.py

```diff
@@ -277,12 +291,11 @@ def PyExec(self):
 
         detID = calWS.column('detid')
 
-        if self.getProperty("Workspace").value is not None:
-            wks_name = self.getProperty("Workspace").value.name()
+        wks_name = "alignedWorkspace"  # workspace whose counts will be DIFC values
```

Comment:
Does this workspace ever get cleaned out? Also, if it is temporary (appears to be) it would be better for the name to be `__alignedWorkspace` so it doesn't automatically show in the data tree in the gui